README.md

```diff
@@ -650,17 +651,74 @@ One of three out comes can occur after the export retried has completed.
 
 ### Secrets
 
-#### Getting Secrets
+#### Configuration
 
-Application Services can retrieve their secrets from the underlying secret store using the [GetSecrets()](#.GetSecrets()) API in the SDK. 
+All instances of App Services share the same database and database credentials. However, there are secrets for each App Service that are exclusive to the instance running. As a result, two separate configuration for secret store clients are used to manage shared and exclusive application service secrets.
+
+The GetSecrets() and StoreSecrets() calls  use the exclusive secret store client to manage application secrets.
+
+An example of configuration settings for each secret store client is below:
+
+```toml
+# Shared Secret Store
+[SecretStore]
+    Host = 'localhost'
+    Port = 8200
+    Path = '/v1/secret/edgex/appservice/'
+    Protocol = 'https'
+    RootCaCertPath = '/tmp/edgex/secrets/ca/ca.pem'
+    ServerName = 'edgex-vault'
+    TokenFile = '/tmp/edgex/secrets/edgex-appservice/secrets-token.json'
+    # Number of attempts to retry retrieving secrets before failing to start the service.
+    AdditionalRetryAttempts = 10
+    # Amount of time to wait before attempting another retry
+    RetryWaitPeriod = "1s"
+
+	[SecretStore.Authentication]
+		AuthType = 'X-Vault-Token'	
+
+# Exclusive Secret Store
+[SecretStoreExclusive]
+    Host = 'localhost'
+    Port = 8200
+    Path = '/v1/secret/edgex/<app service key>/'
+    Protocol = 'https'
+    ServerName = 'edgex-vault'
+    TokenFile = '/tmp/edgex/secrets/<app service key>/secrets-token.json'
+    # Number of attempts to retry retrieving secrets before failing to start the service.
+    AdditionalRetryAttempts = 10
+    # Amount of time to wait before attempting another retry
+    RetryWaitPeriod = "1s"
+
+    [SecretStoreExclusive.Authentication]
+    	AuthType = 'X-Vault-Token'
+```
+
+#### Storing Secrets
+
+##### Secure Mode
+
+When running an application service in secure mode, secrets can be stored in the secret store (Vault) by making an HTTP `POST` call to `http://[host]:[port]/api/v1/secrets`. The secrets are stored and retrieved from from the secret store based on values in the *SecretStoreExclusive* section of the configuration file. Once a secret is stored, only the service that added the secret will be able to retrieve it.  For secret retrieval see [Getting Secrets](#getting-secrets).
```

Comment:
Clarify the uer (http://[host]:[port]/api/v1/secrets) is on the application service.

Reply:
`retrieved from from the` => `retrieved from the`